app/controllers/landing_page_controller.rb

```diff
@@ -20,13 +21,32 @@ def index
     cid = community_id(request)
     version = CLP::LandingPageStore.released_version(cid)
 
-    # TODO Ideally we would do the caching based upon just clp_version
-    # and avoid loading and parsing the (potentially) big structure
-    # JSON.
+    cache_time = APP_CONFIG[:clp_cache_time].to_i.seconds
```

Comment:
Should be a constant. config properties never change dynamically, not even in rails dev mode.